pyaedt/modeler/cad/Modeler.py

```diff
@@ -2773,21 +2773,32 @@ def convert_to_selections(self, object_id, return_list=False):
             return ",".join([str(i) for i in objnames])
 
     @pyaedt_function_handler()
-    def split(self, objects, plane, sides="Both"):
+    def split(self, objects, plane=None, sides="Both", tool=None, split_crossing_objs=False, delete_invalid_objs=True):
         """Split a list of objects.
 
         Parameters
         ----------
         objects : str, int, or list
             One or more objects to split. A list can contain
             both strings (object names) and integers (object IDs).
-        plane : str
+        plane : str, optional
             Coordinate plane of the cut or the Application.PLANE object.
             Choices for the coordinate plane are ``"XY"``, ``"YZ"``, and ``"ZX"``.
+            If neither plane nor tool parameters are provided, the plane tool option is used.
+            The default value is ``"XY"``.
```

Comment:
```suggestion
            Coordinate plane of the cut or the ``Application.PLANE`` object.
            The default is ``None``, in which case the XY coordinate plane is
            used. Options for the coordinate plane are ``"XY"``, ``"YZ"``, and
            ``"ZX"``.  If plane or tool parameters are not provided, the plane
            tool option is used.
```
We try to list/explain the default first and then provide the options in alphabetical order.

Reply:
Hi @PipKat ,
The PR is still in draft so I've made some changes since you commented.
Please take a look now.
Thanks!